recipes/sigmut/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sigmut" %}
+{% set version = "1.0" %}
+{% set sha256 = "414817a3b5c7513d72226060324f093a2bcfcf96e3559a6252a053287028307c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lgpdv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - python >=3.7
+    - matplotlib >=3.1.0
```

Comment:
Please use `matplotlib-base` (also below in the `run:` section). See #19996

Reply:
Thank you very much for your help @marcelm. I will modify the files and make the PR again.